src/main/java/internetshop/dao/jdbc/UserDaoJdbcImpl.java

```diff
@@ -0,0 +1,195 @@
+package internetshop.dao.jdbc;
+
+import internetshop.dao.UserDao;
+import internetshop.exceptions.DataProcessingException;
+import internetshop.lib.Dao;
+import internetshop.model.Role;
+import internetshop.model.User;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+@Dao
+public class UserDaoJdbcImpl extends AbstractDao<User> implements UserDao {
+
+    public UserDaoJdbcImpl(Connection connection) {
+        super(connection);
+    }
+
+    @Override
+    public User create(User user) throws DataProcessingException {
+        user.setToken(getToken());
+        String query = "INSERT INTO users(user_name, user_pass, user_token) VALUES (?, ?, ?);";
+        String query2 = "INSERT INTO users_roles (user_id, role_id) VALUES (?, 2);";
```

Comment:
let's rename these variables and give them more understandable names

Reply:
done